scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/forked/env/ExecuteForkedTaskInsideNewJvm.java

```diff
@@ -77,6 +72,9 @@ public static void main(String[] args) throws Throwable {
         }
 
         fromForkedJVM(args[0]);
+
+        // Call to System.exit is necessary at this point (when the task is finished normally) as the forked JVM can keep alive non-daemon threads
+        System.exit(0);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this call to "exit" or ensure it is really required. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1147)